releasenotes.rb

```diff
@@ -3,9 +3,35 @@
 
 require "thor"
 require "git"
+require "octokit"
+require "logger"
 require "erb"
 require "ostruct"
 
+class GitHubCommitTags
+  attr_reader :client, :repository, :logger
+
+  def initialize(access_token:, repository:, logger: Logger.new(STDERR))
+    @client = Octokit::Client.new(access_token: access_token)
+    @repository = repository
+    @logger = logger
+  end
+
+  def special_deploy_requirements?(sha)
+    search_results = client.search_issues("#{sha} repo:#{repository} is:merged")
+
+    if search_results.total_count != 1
+      logger.warn("SHA #{sha} had #{search_results.total_count} issues, not just one; ignoring.")
```

Comment:
I was actually just going to ask about this. I ran the PR locally and got this warning
```
W, [2016-11-30T15:47:44.378036 #97859]  WARN -- : SHA 0a5f08f15fcfcee2c69334a29d05534f149526a8 had 0 issues, not just one; ignoring.
```

What mean?